### DISCUSSION TITLE
text-ellipsis doesn't work

### DISCUSSION THREAD

**Discussion by davidbourrel** [tailwindlabs/tailwindcss#6592] [Help]
**What version of Tailwind CSS are you using?**

v3.0.6

**What build tool (or framework if it abstracts the build tool) are you using?**

Next.js 12.0.7, postcss 8.4.5, autoprefixer 10.4.0

**What version of Node.js are you using?**

v17.0.0

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows 10

**Describe your issue**

I think there's a problem with **text-ellipsis** class name because it doesn't work at all. 
When I put this one in a tag, Chrome can see the class in the DOM inspector while it doesn't appear in the tab **Styles**.

I also found that text-ellipsis could work with : **overflow-ellipsis**.

was it just a wrong renaming problem or it will be like that in the future ?  

Thanks !

**Comment by adamwathan** [comment]
Hey! It does work:

https://play.tailwindcss.com/wPnRmgsoFg

Going to move this to "Help" since it seems specific to your setup.

**Comment by lukewarlow** [comment]
Do you have a repository showing the issue someone could pull to take a look?

**Comment by Lukmauu** [comment]
You my be using `.inline-flex` with it

**Reply by andrewfrankcare** [reply]
The `flex`class seems incompatible.

**Comment by davidbourrel** [comment]
Sorry, I can't show you :s I did it in another way but it was really strange because if I add the class in the **style** tag, it works xD 

We may close my issue but if I can reproduce this problem, I'll do it.

Thanks all !
